terraform/dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.6-alpine
+
+LABEL maintainer="Julien Pierini <julien.pierini@ekino.com>"
+
+ARG TERRAFORM_VERSION
+
+RUN apk update
```

Comment:
Please merge the `RUN` commands to have only one

Reply:
Maybe we should install the [ci-helper](https://github.com/rande/gitlab-ci-helper) as well like other images?

Reply:
Ci-helper isn't necessary in this image. We do not have to manage build dependencies when deploying with Terraform.